cfme/tests/cli/test_appliance_console_db_restore.py

```diff
@@ -1,10 +1,17 @@
+import re
 from collections import namedtuple
 
 import fauxfactory
 import pytest
 from wait_for import wait_for
 
 from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.fixtures.cli import configure_appliances_ha
+from cfme.fixtures.cli import configure_automatic_failover
+from cfme.fixtures.cli import provider_app_crud
+from cfme.fixtures.cli import reconfigure_primary_replication_node
+from cfme.fixtures.cli import reconfigure_standby_replication_node
+from cfme.fixtures.cli import replicated_appliances_with_providers
```

Comment:
Q: is it not possible to use them as fixtures only?

Reply:
I wouldn't.
1) Although not specified, I think reconfiguring is part of the test steps
2) IIRC there are some tests to be automated that will need different things to be executed prior of configuring/reconfiguring. Perhaps I am not yet used to that but I find composing the fixtures into complex structures more difficult than calling functions. If there is some benefit of having a fixture, I can create a fixture calling this function.